examples/redis-demo/src/test/groovy/com/example/ProxyAwareSpec.groovy

```diff
@@ -1,13 +1,10 @@
 package com.example
 
-import grails.core.support.proxy.ProxyHandler
-
 trait ProxyAwareSpec {
 
-    ProxyHandler proxyHandler
 
     Serializable getEntityId(Object obj) {
-        Serializable identifier = proxyHandler.getIdentifier(obj)
+        Serializable identifier = proxyHandler.unwrapIfProxy(obj)?.ident()
```

Comment:
it also appears that getIdentifier is no longer a valid method so I modified it to this